tests/regression_tests/test_llama2_7b.py

```diff
@@ -50,7 +50,7 @@ def test_finetune_and_eval(self, tmpdir, caplog, monkeypatch):
         runpy.run_path(TUNE_PATH, run_name="__main__")
         eval_cmd = f"""
         tune run eleuther_eval \
-            --config eleuther_eval \
+            --config eleuther_evaluation \
```

Comment:
How was this test working before??

Reply:
It wasn’t, it’s been failing in CI for a while now. But it doesn’t run on PRs or anything, only nightly